components/Form.js

```diff
@@ -0,0 +1,73 @@
+import { useRouter } from "next/router";
+import { styled } from "styled-components";
+
+export default function Form({ submitNewCard, formName }) {
```

Comment:
Does `formName` exist? It doesn't appear to be passed into this component from anywhere. If you `console.log` it here, you may find out that it's `undefined`.

Reply:
cool thanks